testonly/hammer/hammer.go

```diff
@@ -178,6 +178,17 @@ func (c MapConfig) String() string {
 func HitMap(cfg MapConfig) error {
 	ctx := context.Background()
 
+	if cfg.MapID == 0 {
+		// No mapID provided, so create an ephemeral tree to test against.
+		var err error
+		cfg.MapID, err = makeNewMap(ctx, cfg.Admin, cfg.Client)
+		if err != nil {
+			return fmt.Errorf("failed to create ephemeral tree: %v", err)
+		}
+		glog.Infof("testing against ephemeral tree %d", cfg.MapID)
+		defer destroyMap(ctx, cfg.Admin, cfg.MapID)
```

Comment:
Error return value of `destroyMap` is not checked

Reply:
check the return value?

Reply:
Done.